Makefile

```
@@ -214,8 +214,8 @@ jaeger-ui/packages/jaeger-ui/build/index.html:
 
 .PHONY: rebuild-ui
 rebuild-ui:
-	cd jaeger-ui && yarn install --frozen-lockfile && cd packages/jaeger-ui && yarn build
-
+	chmod u+x scripts/rebuild-ui.sh && ./scripts/rebuild-ui.sh
+	
```

just call it with bash
```suggestion
	bash ./scripts/rebuild-ui.sh

```